Extensions/Guardrail+Settings.swift

```diff
@@ -9,7 +9,7 @@
 import HealthKit
 
 public extension Guardrail where Value == HKQuantity {
-    static let suspendThreshold = Guardrail(absoluteBounds: 67...110, recommendedBounds: 74...80, unit: .milligramsPerDeciliter)
+    static let suspendThreshold = Guardrail(absoluteBounds: 67...110, recommendedBounds: 74...80, unit: .milligramsPerDeciliter, startingSuggestion: 80)
```

Comment:
Please have these `startingSuggestion`s reviewed by Product.

Reply:
These starting suggestions do not affect Tidepool Loop, as the onboarding will use initial values from the user's prescription. If Tidepool Loop eventually allows clean slate setup, we will need to tackle providing these. At some point guardrails and initial recommendations will likely diverge between Tidepool Loop and DIY, and we'll need to parameterize it in some fashion.